Tools/NeedleTools.xcodeproj/Basic_Info.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This thing uses Swift Package Manager right ? Why is the project file and such being included ?

Reply:
I pulled in the Swift Package Manager for the commandline utilities. But it also uses Carthage to pull in OctoKit, the GitHub SDK. I can remove that for now, since I couldn't get the GitHub API to work anyways.